packages/trpc/server/routers/viewer/workflows/activateEventType.handler.ts

```diff
@@ -216,14 +231,9 @@ export const activateEventTypeHandler = async ({ ctx, input }: ActivateEventType
       // activate workflow and schedule reminders for existing bookings
       const bookingsForReminders = await prisma.booking.findMany({
         where: {
-          OR: [
-            { eventTypeId },
-            {
-              eventType: {
-                parentId: eventTypeId,
-              },
-            },
-          ],
+          eventTypeId: {
+            in: activeOn,
```

Comment:
cc @keithwillcode - addresses earlier (valid) concern - not as big an issue as below but still an avenue I could optimise.